Module/uti/dan_kin_boots.uti.json

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
The name of the item is "Boots" but the perk needed is "Leggings". Please correct so it's consistently named.